tools/clang/tools/dxil2spv/lib/dxil2spv.cpp

```diff
@@ -147,26 +148,42 @@ void Translator::Run() {
   }
 
   // Contsruct the SPIR-V module.
-  std::vector<uint32_t> m = spvBuilder.takeModuleForDxilToSpv();
+  std::vector<uint32_t> spirvModule = spvBuilder.takeModuleForDxilToSpv();
 
   // Validate the generated SPIR-V code.
   std::string messages;
-  if (!spirvToolsValidate(&m, &messages)) {
+  if (!spirvToolsValidate(&spirvModule, &messages)) {
     emitError("Generated SPIR-V is invalid: %0") << messages;
   }
 
-  // Disassemble SPIR-V for output.
-  std::string assembly;
-  spvtools::SpirvTools spirvTools(SPV_ENV_VULKAN_1_1);
-  uint32_t spirvDisOpts = (SPV_BINARY_TO_TEXT_OPTION_FRIENDLY_NAMES |
-                           SPV_BINARY_TO_TEXT_OPTION_INDENT);
+  // If output file not specified, disassemble SPIR-V for stdout.
+  std::string outputFile = ci.getFrontendOpts().OutputFile;
+  if (outputFile.empty()) {
+    std::string assembly;
+    spvtools::SpirvTools spirvTools(SPV_ENV_VULKAN_1_1);
```

Comment:
I see `instance.getCodeGenOpts().SpirvOptions.targetEnv = "vulkan1.0";` above. If we use `SPV_ENV_VULKAN_1_1` for the default vulkan env, it looks like a mismatch. Maybe `SPV_ENV_VULKAN_1_0` by default?

Reply:
Done!